### PR TITLE
Deleting vehicles server-side

### PR DESCRIPTION
**Pull request by boostless**

### CODE REVIEW

File: client/client.lua [hunk 1]
```diff
@@ -519,14 +519,13 @@ AddEventHandler('luke_garages:StoreVehicle', function(target)
     ESX.TriggerServerCallback('luke_garages:CheckOwnership', function(doesOwn)
         if doesOwn then
             if type(doesOwn) == 'table' then return ESX.ShowNotification("You can't store your job vehicle here") end
-            ESX.Game.DeleteVehicle(vehicle)
 
             TriggerServerEvent('luke_garages:ChangeStored', vehPlate, true, currentGarage.zone.name)
 
-            TriggerServerEvent('luke_garages:SaveVehicle', vehProps, health, vehPlate)
+            TriggerServerEvent('luke_garages:SaveVehicle', vehProps, health, VehToNet(vehicle))
```
Comment (LukeWasTakenn) [inline client/client.lua:525]: Why remove the vehPlate param?

File: server/server.lua [hunk 2]
```diff
@@ -123,7 +123,8 @@ AddEventHandler('luke_garages:ChangeStored', function(plate, stored, garage)
 end)
 
 RegisterNetEvent('luke_garages:SaveVehicle')
-AddEventHandler('luke_garages:SaveVehicle', function(vehicle, health, plate)
+AddEventHandler('luke_garages:SaveVehicle', function(vehicle, health, plate, ent)
```
Comment (LukeWasTakenn) [inline server/server.lua:126]: `ent` will be nil because you aren't sending the vehPlate param anymore

**Review by LukeWasTakenn — CHANGES_REQUESTED** [review]

**Comment by LukeWasTakenn** [comment]
Any update on this?

**Comment by boostless** [comment]
Going to push an update tomorrow

**Comment by LukeWasTakenn** [comment]
> Going to push an update tomorrow. Utilizing new nh-context, vehicle deletion

I don't want new nh-context, I was actually considering switching over to zf-context

**Comment by boostless** [comment]
Could do that cause i thinks is just an export change

**Comment by LukeWasTakenn** [comment]
> Could do that cause i thinks is just an export change

I could be wrong but iirc it actually uses the exact same event and export names so it should just work out of the box...?

**Comment by boostless** [comment]
Added missing vehicle plate

**Comment by LukeWasTakenn** [comment]
Thank you

**Pull request merged**